Code/Framework/AtomCore/Tests/InstanceDatabase.cpp

```diff
@@ -254,10 +310,12 @@ namespace UnitTest
         // Used to detect a deadlock.  If we wait for more than 10 seconds, it's likely a deadlock has occurred
         while (threadCount > 0 && !timedOut)
         {
+            size_t durationSecondsRoundedUp = static_cast<size_t>(std::ceil(durationSeconds + 1.0f));
```

Comment:
If you want to round up, shouldn't it be "floor(duration+1)" not "ceil(duration+1)"?

Reply:
Should have just been ceil. Will fix.